Makefile.linux

```diff
@@ -56,8 +60,6 @@ LINUX_FLAGS += 	-fno-asynchronous-unwind-tables \
 		-fno-stack-protector \
 		-ffunction-sections \
 		-fdata-sections \
-		-fno-defer-pop \
-		-Wno-pointer-sign \
```

Comment:
They can be removed. The no-defer-pop gets disabled with any optimizations (-O_) and the no-pointer-sign disables warnings for pointer conversions.